pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -71,7 +71,6 @@
 
   protected final File _tempDir = new File(FileUtils.getTempDirectory(), getClass().getSimpleName());
   protected final File _avroDir = new File(_tempDir, "avroDir");
-  protected final File _preprocessingDir = new File(_tempDir, "preprocessingDir");
```

Comment:
Hmm.. I remember we had a Hadoop integration test that tests the SegmentPreprocessing Job which uses this param `_preprocessingDir ` but couldn't find it in the code though.